pages/blog/new-asyncapi-tools-page.md

```diff
@@ -0,0 +1,136 @@
+
+![Dashboard UI](/img/posts/new-asyncapi-tools-page/dashboard-preview.webp)
+
+The Dashboard consists of various features like the overall information about the tools inside ToolCard, filters to sort out the tools according to your needs, and search for tools specific to its name and type.
+
+## Tool Card
+
+Taking the Tools Card under consideration, you have following sections to detail about a Tool:
+
+![Tools Card](/img/posts/new-asyncapi-tools-page/tool-preview.webp)
+
+<ol>
+  <li> Specifies the Name of the Tool.</li>
+  <li> Specifies whether the Tool is Free to use or requires commercial access to use it.</li>
+  <li> Specifies the Description of the Tool. Clicking on the `Show More` button will open a small box to show full description of the Tool.</li>
+  <li> Specifies the Primary Language of the Tool in which it is built.</li>
+  <li> Specifies the list of Technologies used to create the Tool.</li>
+  <li> <b>View on Github</b> Button that directs the user to the Github repository of the Tool.</li>
+  <li> <b>Visit Website</b> Button that directs the user to the official website of the Tool.</li>
+  <li> <b>Visit Docs</b> Button that directs the user to the official Docs of the Tool.</li>
+</ol>
+
+## Filters for Tools
+
+The Dashboard also contain various filters to search for a specific Tool according to your needs. You can now search according to the Name, Languages, Technologies, Categories, etc.. as provided in respective dropdowns and radio buttons. Filters are mostly divided in 2 sections - Filter Menu and Search bar. The use of each menu is explained below under respective subheadings.
+
+![Filters Preview](/img/posts/new-asyncapi-tools-page/filters.webp)
+
+### Filter Menu
+
+Taking the Tools Filter Menu under consideration, you have following filters to search a Tool:
+
+![Filter Menu](/img/posts/new-asyncapi-tools-page/filter-menu.webp)
+
+<ol>
+  <li> This radio filter button allows to search for either free tools available on internet or Github, or search for the Paid Tools.</li>
+  <li> The slider filter button allows to search for the tools owned or maintained by AsyncAPI organization. </li>
+  <li> The <b>Language Filter</b> allows you to select the languages from the dropdown on which you want to search for Tools. </li>
+  <li> The <b>Technology Filter</b> allows you to select the technologies from the dropdown on which you want to search for Tools. </li>
+  <li> The <b>Clear Filters</b> button allows you to clear all filters inside the Filters Modal and will show you all the tools of selected categories (or all categories).</li>
+</ol>
+
+Do remember to click on `Apply Filter` Button before closing the `Filters` Modal so that filters will be applied on the Dashboard.
```

Comment:
```suggestion
Do remember to click on the `Apply Filter` Button before closing the `Filters` Modal so that filters will be applied on the Dashboard.
```